editoast/src/views/scenario/mod.rs

```diff
@@ -45,11 +48,15 @@ crate::routes! {
             get,
             delete,
             patch,
+            &macro_nodes,
         },
     },
 }
 
 editoast_common::schemas! {
+    MacroNodeForm,
+    MacroNodeResponse,
+    MacroNodeListResponse,
```

Comment:
Can these be included in `views/scenario/macro_nodes.rs` to preserve locality and avoid unnecessary `pub`s?

Reply:
If I do the `editoast_common::schemas` in `macro_nodes.rs` (instead of in `mod.rs`) the generated `openapi.yml` has no definition for those objects (ie. `/components/schemas/MacroNodeResponse` is missing).

Reply:
Yes because all `schema!` does in the end is generate a `fn schemas()`. You can reference it in the openapi in `mod.rs` like so:

https://github.com/OpenRailAssociation/osrd/blob/2449281e9e9ee5c3e60f60597a1e3c81b2c49a90/editoast/src/views/mod.rs#L92-L118